ADVANCED.md

```diff
@@ -1158,7 +1159,7 @@ For example, if you add a `search_fields` configuration option, it will control
 
 ## Global filters
 
-Using Query Config, it is possible to create "Global" filters. "Global filters" are filters that are added to *every* query. The user has no control over whether or not this filter is added or removed, it doesn't show up in the query string, and is completely transparent to the user. It is applied IN ADDITION to filters which the user applies.
+Using Query Config, it is possible to create "Global" filters. "Global filters" are filters that are added to _every_ query. The user has no control over whether or not this filter is added or removed, it doesn't show up in the query string, and is completely transparent to the user. It is applied IN ADDITION to filters which the user applies.
```

Comment:
Some of these changes were just from the autoformatter. The only thing I did was add `thumbnailField` above.